streams/src/main/java/org/apache/kafka/streams/processor/internals/DefaultKafkaClientSupplier.java

```diff
@@ -27,6 +25,8 @@
 import org.apache.kafka.common.serialization.ByteArraySerializer;
 import org.apache.kafka.streams.KafkaClientSupplier;
 
+import java.util.Map;
```

Comment:
nit: revert this file back to what it was originally as the only thing that has changed now is the order of the imports